src/docs/product/releases/health/index.mdx

```diff
@@ -4,6 +4,8 @@ sidebar_order: 50
 redirect_from:
   - /workflow/releases/health/
   - /product/releases/health/crash/
+  - /product/releases/health/</small>
+  - /product/releases/health/crash
```

Comment:
```suggestion
  - /product/releases/health/</small>
  - /product/releases/health/crash
```
/product/releases/health/crash - this one already redirects. 
Is this correct? /product/releases/health/</small>